build.gradle

```
@@ -600,15 +600,10 @@ repositories {
         }
         maven {
             name = "ic2"
-            url = "https://maven.ic2.player.to/"
-            metadataSources {
-                mavenPom()
-                artifact()
+            url = getURL("https://maven.ic2.player.to/", "https://maven2.ic2.player.to/")
+            filter {
```

:woman_facepalming: I copied the wrong line above
```suggestion
            content {
```